floyd/cli/data.py

```diff
@@ -141,6 +141,66 @@ def clone(id):
                               delete_after_untar=True)
 
 
+@click.command()
+@click.argument('id', nargs=1)
+def listfiles(id):
+    """
+    List files in the given dataset
+    """
+
+    data_source = DataClient().get(normalize_data_name(id, use_data_config=False))
```

Comment:
you probably want to set `use_data_config` to True so the client can infer dataset names like `1` or  `foo/1` based on data config to save user typing.

Reply:
actually, I take it back. it's ok to set use_data_config to False for now. I have a better idea on how to handle this going forward.